### DISCUSSION TITLE
How to slice an existing universe at specified timeframe to create new universe?

### DISCUSSION THREAD

**Discussion by rayanc72** [MDAnalysis/mdanalysis#4388] [Q&A] [answered]
Hi all, I would like to create smaller universes of different timeframes (e.g., 2 ps to 5 ps, 3 ps to 6 ps, and so on from an existing universe of 0 ps to 20 ps). What is the best way to achieve this only using the existing universe and without reading or writing any files?

**Comment by IAlibay** [comment]
@rayanc72 Do you mean that you want slices of your trajectory?

Would it work for you to just iterate your analysis over the specific sub sections of the trajectory as needed?

**Reply by rayanc72** [reply]
Thanks! Yes, that would work too. However, I could not find the syntax for that in the documentation. Specifically, I'm trying to use rdf.InterRDF_s at different timeframes. Do you have any suggestions on how to feed in specific sub-sections of the trajectory to this class?

**Reply by IAlibay** [reply, marked as answer]
@rayanc72 I believe that this class uses our standard AnalysisBase class type. To run your analysis over specific frames you can pass the `start`, `stop`, `step` arguments to the `run` method: https://docs.mdanalysis.org/stable/documentation_pages/analysis/base.html#example-of-using-a-standard-analysis-tool

There's also a `frames` argument you can use to pass through a list of frame indices you want to use it that's more useful (it's detailed a bit lower in the above link).

**Reply by rayanc72** [reply]
Thank you! That works! 😅
